apps/mobile/src/libs/components/expired-tasks-modal/expired-tasks-modal.tsx

```diff
@@ -0,0 +1,132 @@
+import React from "react";
+
+import {
+	Button,
+	ScrollView,
+	TaskCard,
+	Text,
+	View,
+} from "~/libs/components/components";
+import { NumericalValue, TaskStatus } from "~/libs/enums/enums";
+import {
+	useAppDispatch,
+	useCallback,
+	useMemo,
+	useWindowDimensions,
+} from "~/libs/hooks/hooks";
+import { globalStyles } from "~/libs/styles/styles";
+import { type TaskDto } from "~/packages/tasks/tasks";
+import { actions as taskActions } from "~/slices/task/task";
+
+import { styles } from "./styles";
+
+type Properties = {
+	tasks: TaskDto[];
+};
+
+const ExpiredTasksModal: React.FC<Properties> = ({ tasks }) => {
+	const { width } = useWindowDimensions();
+
+	const scrollViewWidth = useMemo(() => tasks.length * width, [tasks, width]);
```

Comment:
```suggestion
	const scrollViewWidth = useMemo(() => {
	  return tasks.length * width
	}, [tasks, width]);
```
Let's make it more readable